etc/kayobe/stackhpc-monitoring.yml

```diff
@@ -9,3 +9,13 @@
 alertmanager_low_memory_threshold_gib: 5
 
 ###############################################################################
+
+###############################################################################
+# Exporter configuration
+
+# Whether the OpenStack Capacity exporter is enabled.
+# Enabling this flag will result in HAProxy configuration being templated during
+# deployment.
+stackhpc_enable_os_capacity: false
```

Comment:
cool, we are off by default now, thank you.

Reply:
Right now the deployment requires manual intervention, it would be cool to get it automated in the future.